contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+# Proposal to Standardize Development Process
```

Comment:
This title seems a little overarching.  Should we instead focus on proposals for major efforts/enhancements?  Not all development is going to funnel through this.

Reply:
I believe that the vast majority of development should work through a centralized process until we find concrete examples of work which is poorly served. Were we to discover that large classes of work are poorly described by the proposed process I think we should revisit the process. I believe process should serve the people and if we discover that a process is no longer working we should change it.